vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -29,12 +29,13 @@ import (
 	"k8s.io/klog"
 )
 
-const (
+var (
 	// Ignore change priority that is smaller than 10%.
-	defaultUpdateThreshold = 0.10
+	defaultUpdateThreshold = flag.Float64("priority-update-threshold", 0.10, "Ignore change priority that is smaller this value")
```

Comment:
Please make the flag description more descriptive.

Ignore updates that have priority lower than the value of this flag.

Short explanation of what is update priority would be useful too.